components/WhatsHappening.vue

```diff
@@ -8,6 +8,7 @@
         <p class="description">
           April 22 will be the 50th anniversary of Earth Day.
           That’s why we’re launching a massive livestream from April 22-24 where millions of people can tune in with activists, celebrities, musicians, and more in an epic celebration of our planet.
+          In the midst of an unprecented global pandemic, it's more important than ever to unite and support each other during this crisis.
```

Comment:
Asking some others about this one - it's good, but makes the section too long.

Reply:
okay, we aligned on the following changes:

```In the midst of this global crisis, it's more important than ever to unite and support each other. Together, we'll support the relief effort and fight for a better world on the other side of the pandemic.

Here are all the ways you can help.```

This would replace lines 11,12 and 13. If you can makes those changes and re-push for me to check, I'll merge it down!